webpack.common.js

```diff
@@ -10,9 +10,7 @@ module.exports = {
             template: path.resolve(__dirname, "src", "index.html"),
         }),
         new MiniCssExtractPlugin(),
-        new MonacoWebpackPlugin({
-            languages: ["ascii", "ini"],
-        }),
+        new MonacoWebpackPlugin({}),
```

Comment:
What's the reason for this change? We originally specified the required languages to shrink the build (without specifying the languages, @junhaoliao measured the build is twice as large).

Reply:
Sorry this was my oversight. @Henry8192 As you integrate above suggestion, you may also do a size comparison on your end:
```
cd path/to/project/root

# Linux: build bundle with profiling option
webpack --profile --json > stats.json
# OR Windows: build bundle with profiling option
webpack --profile --json | Out-file 'stats.json' -Encoding OEM

# now you can check size of the `dist` directory generated under project root

# you can run webpack-bundle-analyzer to show the size of each component
npx webpack-bundle-analyzer stats.json
```
Reference: https://www.npmjs.com/package/webpack-bundle-analyzer

Things to try as you optimize the bundle size:
1. Specify `features: []` in `options` passed to `new MonacoWebpackPlugin()`.
2. (This shall go into a separate PR) Try https://webpack.js.org/plugins/compression-webpack-plugin/ with algorithms `gzip` / `br`.